lib/handlers/ortc/edgeUtils.js

```diff
@@ -75,6 +75,11 @@ exports.mangleRtpParameters = function(rtpParameters)
 			delete codec.channels;
 		}
 
+		if(codec.mimeType && !codec.name)
+		{
+			codec.name = codec.mimeType.split('/')[1]
```

Comment:
We don't use `codec.name` at all in the code but just  `codec.mimeType`. So this is not needed.